### ISSUE TITLE
multiple repository

### ISSUE THREAD

**Issue by shaxi007** [microsoft/vscode#190007]
Type: <b>Bug</b>

multiple repositories

VS Code version: Code 1.81.0 (Universal) (6445d93c81ebe42c4cbd7a60712e0b17d9463e97, 2023-08-02T12:40:02.782Z)
OS version: Darwin x64 22.6.0
Modes:

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-8850H CPU @ 2.60GHz (12 x 2600)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>metal: disabled_off<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: enabled|
|Load (avg)|11, 8, 6|
|Memory (System)|16.00GB (2.54GB free)|
|Process Argv|--crash-reporter-id 41d92bfa-b385-4431-89ed-98af14987d4f|
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (9)</summary>

Extension|Author (truncated)|Version
---|---|---
vscode-eslint|dba|2.4.2
githistory|don|0.6.20
gitlens|eam|2023.8.805
dotenv|mik|1.0.1
material-icon-theme|PKi|4.29.0
vscode-protolint|ple|0.8.0
LiveServer|rit|5.7.9
code-spell-checker|str|2.20.5
vscode-proto3|zxh|0.5.5

(1 theme extensions excluded)

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
vsreu685:30147344
python383cf:30185419
vspor879:30202332
vspor708:30202333
vspor363:30204092
vslsvsres303:30308271
vserr242:30382549
pythontb:30283811
vsjup518:30340749
pythonptprofiler:30281270
vshan820:30294714
vstes263cf:30335440
vscod805cf:30301675
binariesv615:30325510
bridge0708:30335490
bridge0723:30353136
vsaa593:30376534
pythonvs932:30410667
py29gd2263cf:30792227
vsclangdc:30486549
c4g48928:30535728
dsvsc012cf:30540253
pynewext54:30695312
azure-dev_surveyone:30548225
vsccc:30803844
282f8724:30602487
f6dab269:30613381
showlangstatbar:30737416
vsctsb:30748421
03d35959:30757346
pythonfmttext:30731395
pythoncmvfstrcf:30756944
fixshowwlkth:30771522
showindicator:30805244
pythongtdpath:30769146
i26e3531:30792625
gsofa:30804715
pythonnosmt12:30797651
pythonidxpt:30805730
pythonnoceb:30805159
e537b577:30795824
h0f32768:30792100
dsvsc013:30795093
dsvsc014:30804076

```

</details>

**Comment by VSCodeTriageBot** [comment]
Thanks for creating this issue! We figured it's missing some basic information or in some other way doesn't follow our [issue reporting guidelines](https://aka.ms/vscodeissuereporting). Please take the time to review these and update the issue.

Happy Coding!

**Comment by Aking8089** [comment]
Same issue as of yesterday as well no extensions are working and none of my code is being saved it all got deleted, It also created another vscode somehow it started first with multiple repo's just a few actually.

**Comment by lszomoru** [comment]
@Aking8089, could you please file a separate issues as what you are describing does not seem to be relevant to this issue. Thank you very much in advance!

**Comment by VSCodeTriageBot** [comment]
This issue has been closed automatically because it needs more information and has not had recent activity. See also our [issue reporting](https://aka.ms/vscodeissuereporting) guidelines.

Happy Coding!

**Comment by Aking8089** [comment]
I was hacked how do I resolve this immediately just recently found out that's why I had 2 unworkable visual studio codes and completely different and different extensions. Sometimes prior I kept notice new extensions being added almost like they were in sync with each other but for an entire week and half I could not access them actual longer probably almost 3 weeks. I lost python at one point as well. Plus some of my extensions. Another profile was added. I am trying to handle this now but the bot isn't much help. For security. I need to know what I should do because at this point they have access to everything due to having all my passwords and usernames etc. They completely deleted c drive and one drive then one note I was able to recover most of it.
